src/components/AttachmentModal.tsx

```diff
@@ -510,56 +538,49 @@ function AttachmentModal(props) {
                     )}
                 </GestureHandlerRootView>
             </Modal>
-            {!props.isReceiptAttachment && (
+            {!isReceiptAttachment && (
                 <ConfirmModal
-                    title={attachmentInvalidReasonTitle ? translate(attachmentInvalidReasonTitle) : ''}
+                    title={attachmentInvalidReasonTitle ? translate(attachmentInvalidReasonTitle as TranslationPaths) : ''}
                     onConfirm={closeConfirmModal}
                     onCancel={closeConfirmModal}
                     isVisible={isAttachmentInvalid}
-                    prompt={attachmentInvalidReason ? translate(attachmentInvalidReason) : ''}
+                    prompt={attachmentInvalidReason ? translate(attachmentInvalidReason as TranslationPaths) : ''}
                     confirmText={translate('common.close')}
                     shouldShowCancelButton={false}
                 />
             )}
 
-            {props.children &&
-                props.children({
+            {children &&
+                typeof children === 'function' &&
+                children({
```

Comment:
Can it be simplified?
```suggestion
            {children?.({
```